### ISSUE TITLE
Markdown doesn't render on additional pages

### ISSUE THREAD

**Issue by ksilz** [StefMa/hugo-fresh#106]
I used your theme for a Hugo site. I added a markdown page to `content`. Its content isn't translated from Markdown into HTML (like `###` for headings or `- blah` for lists. To get any sort of styling, I have to embed HTML into the page, like in the `exampleSite` of the theme.

I'm using the latest version of Hugo (v0.63.2/extended) on the latest version of macOS Catalina (10.15.3). Here's the beginning of my `config.yaml`:

````
theme: fresh
enableEmoji: true
enableRobotsTXT: true
enableGitInfo: true

googleAnalytics: # Put in your tracking code without quotes like this: UA-XXXXXX...
disableKinds:
  - taxonomy
  - taxonomyTerm
markup:
  goldmark:
    renderer:
      unsafe: true # Allows you to write raw html in your md files

params:
````

And here's the beginning my `content/resume.md` page :

````
---
title: "My Resume"
date: 2020-01-31T06:00:00+00:00
draft: false
include_footer: true
sidebar: false
---
````

**Comment by dpaqfb** [comment]
@StefMa - thanks for this neat theme. Any update on @ksilz 's issue ?  I had added a single post here: https://github.com/frizbane/static-hugo-fresh-default after following the setup steps in README. Any help is appreciated.

**Comment by lucperkins** [comment]
Fixed in 995b901e8c0f5717dae0024d28262506e0acc59b

**Comment by dpaqfb** [comment]
Brilliant  - that works. Thanks @lucperkins
